src/main/java/ArraySortedBag.java

```diff
@@ -109,7 +109,7 @@ private int findInsertIndex(T element) {
             }
             searchIndex++;
         }
-        return searchIndex;
+        return rear;
```

Comment:
They'll be the same, but I think `rear` makes it clearer what this would mean if we get to this line (search with no spot found, so it goes to the end).

Reply:
Reasonable change, a `// the new element must belong at the end` would make it clearer too.